### DISCUSSION TITLE
Popconfirm whith Segmented-component

### DISCUSSION THREAD

**Discussion by asalekseev** [ant-design/ant-design#39987] [Q&A] [answered]
Hi!
I use Popconfirm whith Segmented-component. 
How can I make the сonfirm-panel appear above the selected item and not in fixed positions?

![Screenshot_3](https://user-images.githubusercontent.com/90420814/210388682-05199f43-15e9-4493-809f-2e68366b5046.png)

**Comment by zombieJ** [comment, marked as answer]
Segmented `options` accept `label` as ReactNode. Just wrapper confirm on the `label` node.
